tests/libcxx-backports/mdspan/mdspan.index_operator.pass.cpp

```diff
@@ -38,6 +39,9 @@
 #include "../ConvertibleToIntegral.h"
 #include "../CustomTestLayouts.h"
 
+// This test uses the bracket operator, but its not something we have at configure time
+#if MDSPAN_USE_BRACKET_OPERATOR
+
 // Clang 16 does not support argument packs as input to operator []
 #if defined(__clang_major__) && __clang_major__ < 17
```

Comment:
Is this covered by `MDSPAN_USE_BRACKET_OPERATOR`?

Reply:
no, and it was there before.

Reply:
more specifically, clang 16 and 15 can support most uses of bracket operator but not argument packs.

Reply:
Ok I misunderstood what that was there for